Frameworks/CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -53,26 +55,12 @@
 #include <Foundation/NSTimeZone.h>
 #include <Foundation/NSDate.h>
 
-struct tzhead {
```

Comment:
We could also just make this a new file and exclude the old one from the build locally!